nautobot/extras/models/jobs.py

```diff
@@ -784,7 +795,9 @@ def enqueue_job(
         )
 
         if task_queue is None:
-            task_queue = settings.CELERY_TASK_DEFAULT_QUEUE
+            task_queue = (
+                job_model.default_job_queue.name if job_model.default_job_queue else settings.CELERY_TASK_DEFAULT_QUEUE
+            )
```

Comment:
default_job_queue is non-nullable now, so:

```suggestion
            task_queue = job_model.default_job_queue.name
```